rfc077-mode-agnostic-data-collection.md

```diff
@@ -525,6 +445,7 @@ The Run End Schema will be used by Chef Client to notify the data collection ser
         "expanded_run_list",
         "message_type",
         "message_version",
+        "node",
```

Comment:
The expectation is a converge is changing the state of a node (or at least asserting the state is what it currently is), and sending the node along with the converge-is-completed message is like handing over proof of that converge. I'm actually bummed we didn't do this initially :)